Cargo.toml

```diff
@@ -12,7 +12,16 @@ libc = "0.2.152"
 # Default enable napi4 feature, see https://nodejs.org/api/n-api.html#node-api-version-matrix
 napi = { version = "2.12.2", default-features = false, features = ["napi4"] }
 napi-derive = "2.12.2"
-nix = { version = "0.29.0", features = ["fs", "term", "poll"] }
+nix = { version = "0.29.0", features = ["fs", "term", "poll", "signal", "ptrace"] }
+
+[target.'cfg(target_os = "linux")'.dependencies]
+anyhow = "1.0.94"
```

Comment:
@cbrewster what's the latest about using anyhow? i kinda want to drop it, but i also like `.context()`/`.with_context()` :'(

Reply:
I think its fine to use in this context!